services/QuillLMS/app/workers/sync_school_data_from_url_worker.rb

```diff
@@ -48,9 +53,9 @@ def build_attribute_hash(school_data)
       latitude: school_data['latitude'].to_f,
       ulocal: school_data['urban_centric_locale'],
       fte_classroom_teacher: school_data['teachers_fte'].to_i,
-      free_lunches: school_data['enrollment'].to_i > 0 ? (school_data['free_or_reduced_price_lunch'].to_i / school_data['enrollment'].to_f * 100).to_i : nil, # we receive free_or_reduced_price_lunch as an integer but want to store it as a percentage,
-      direct_certification: school_data['enrollment'].to_i > 0 ? (school_data['direct_certification'].to_i / school_data['enrollment'].to_f * 100).to_i : nil, # we receive direct_certification as an integer but want to store it as a percentage
-      total_students: school_data['enrollment'],
+      free_lunches: free_or_reduced_price_lunch && enrollment.to_i > 0 ? (free_or_reduced_price_lunch.to_i / enrollment.to_f * 100).to_i : nil,
```

Comment:
[optional] that these two expressions are borderline ready for their own methods